src/components/ArtObjectPageComponents/PanelDetails/index.jsx

```diff
@@ -182,49 +191,70 @@ class Image extends Component {
       isLoaded,
       object,
       activeImageIndex,
-      setActiveImageIndex
+      setActiveImageIndex,
+      renditions
     } = this.props;
+
+    // This indicates that there was an error with rendering the Zoom component
     const { didCatchFailure } = this.state;
-    
+    const showZoomImageView = Boolean(!didCatchFailure && object.id && activeImageIndex === 0);
+    const renditionsExist = renditions?.length > 0;
+
     let additionalStyle = {};
-    if (!didCatchFailure) {
+    let imageUrlToRender = '';
+
+    // If we encountered failure during rendering of the Zoom component, we'll hide it
+    // Additionally, if the user clicked on an image from the rendition thumbnails, we'll render it instead
+    if (!didCatchFailure && activeImageIndex === 0) {
       additionalStyle = { ...additionalStyle, display: 'none' };
     };
 
+    // We'll render the image from the object itself 
+    if (activeImageIndex === 0) {
+      imageUrlToRender = object.imageUrlLarge
+    }  // Otherwise, one of the renditions was clicked. So we need to render that image
+    else {
+      const imageThumbnailPreview = renditions[activeImageIndex].proxies.find((proxy) => {
+        return proxy.name === 'Preview'
+      });
+      imageUrlToRender = `https://barnesfoundation.netx.net${imageThumbnailPreview.file.url}/`;
+    }
+
+
     return (
       <div>
         <div className='image-art-object'>
-          {(!didCatchFailure && object.id) &&
+          {(showZoomImageView) &&
             <Zoom
               id={object.id}
               catchFailureInViewer={this.catchFailureInViewer}
             />
           }
           <img
-            aria-hidden='true'
+            aria-hidden={showZoomImageView.toString()}
             className='image-art-object__img'
-            src={object.imageUrlLarge}
+            src={imageUrlToRender || object.imageUrlLarge}
             alt={object.title}
             onLoad={onLoad}
             style={{ ...additionalStyle }}
             ref={ref => this.ref = ref}
           />
-          {/** Uncomment this once we have thumbnail data. */}
-          {/* {isLoaded && <div className='image-art-object__button-group'>
+          {/** Renders the left/right navigation arrows when renditions exist */}
+          {isLoaded && renditionsExist && <div className='image-art-object__button-group'>
             <button
               className='btn image-art-object__arrow-button image-art-object__arrow-button--left'
               onClick={() => setActiveImageIndex(activeImageIndex - 1)}
             >
-              <Icon classes='image-art-object__arrow' svgId='-caret-left'/>
+              <Icon classes='image-art-object__arrow' svgId='-icon_arrow-left'/>
             </button>
-            <span className='image-art-object__counter'>{activeImageIndex + 1} / {STATIC_IMAGE_COUNT}</span>
+            <span className='image-art-object__counter'>{activeImageIndex + 1} / {renditions.length}</span>
```

Comment:
We'll only render the left/right caret icons when there's renditions to flip through.